privaterelay/templates/includes/premium-onboarding.html

```diff
@@ -59,8 +59,11 @@ <h1>{% ftlmsg 'multi-part-onboarding-premium-get-domain' %}</h1>
                         {% endif %}
                         
                         <div class="c-premium-onboarding-action-completed {% if user_profile.subdomain %} is-visible {% endif %}">
-                            <img class="margin-right" src="/static/images/icon-check.svg" alt="">
-                            <samp class="js-premium-onboarding-domain-registration-preview">{{ user_profile.custom_domain }}</samp>
+                            <div class="c-premium-onboarding-subdomain-set-module">
+                                <p class="c-premium-onboarding-label">{% ftlmsg 'profile-label-domain' %}</p>
+                                <p class="c-premium-onboarding-subdomain">@{{ user_profile.subdomain }}</p>
+                                <span class="c-premium-onboarding-mozmail">.mozmail.com</span>
+                            </div>
```

Comment:
Ah, I'm running into another error, and I see why you're not running into it (and not in the other one either): they only occur when you claim a subdomain during the onboarding, not if you already have a subdomain before starting the onboarding. That's presumably also why there was JavaScript to set the subdomain in there, rather than just in the Django template.

This is what it looks like for me after registering a subdomain:

![image](https://user-images.githubusercontent.com/4251/152343609-8aa9c12f-9f41-41dd-a7ee-78131e0859d6.png)

The reason for that is that at the time the back-end serves up this page, `user_profile.subdomain` is not set yet. And when the subdomain is registered, the page is not reloaded, so the contents of this element is still `@None.mozmail.com` when it is made visible.

Reply:
@Vinnl  Huh weird, wouldn't it show this:
![image](https://user-images.githubusercontent.com/13066134/152404018-82171f5f-c079-41f3-9056-c32013ef66e6.png)

if a user does not have a subdomain set yet?

Reply:
Ohh! I understand now, thank you so much. I re-added the Javascript that I deleted as well as the `.js-premium-onboarding-domain-registration-preview' class into the HTML. Would this work now?